docs/reference-architectures/data/monitoring.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Solution Accelerator for Monitoring Azure Databricks
+titleSuffix: Azure Solution Accelerators
+description: A scala library to deploy to a Databricks cluster to enable monitoring of metrics and logging data in Azure Log Analytics
+author: petertaylor9999
+ms.date: 01/28/2019
+ms.topic:
+ms.service:
+ms.subservice:
+---
+
+# Solution Accelerator: Monitoring Azure Databricks using Azure Log Analytics
+
+Azure Databricks is a fast, easy and collaborative Apache Spark–based analytics service that makes it easy to rapidly develop and deploy big data analytics and artificial intelligence (AI) solutions. Monitoring is a critical part of any production-level solution, and Azure Databricks offers robust functionality for monitoring metrics, streaming query event information, and application logging data.
+
+Databricks is a flexible and extensible service that supports delivery of metrics and logging data to many different logging services. This solution accelerator extends the core monitoring functionality of Azure Databricks to support Azure Log Analytics.
+
```

Comment:
Maybe, simplify these paragraphs to the essentials. Here is what I get:
1. Azure Databricks develops and deploys big data analytics and AI solutions.
2. It needs Spark.
3. It gives monitoring metrics, streaming query event information, and application logging data.
4. Delivers to LA.

Then maybe introduce the purpose of the solution accelerator?